core/tests/protractor_utils/general.js

```diff
@@ -130,7 +130,21 @@ var scrollElementIntoView = function(elementToScrollTo) {
   }, elementToScrollTo);
 };
 
-
+// Checks no untranslated values are shown inside the given selector.
+var ensurePageHasNoTranslationIds = function(selector) {
+  element(by.css('.oppia-base-container')).then(function(value) {
+    // The use of the InnerHTML is hacky, but is faster than checking each
+    // individual component that contains text.
+    value.getInnerHtml().then(function(promiseValue) {
+      // First remove all the attributes translate and variables that are
+      // not displayed
+      var reTranslateAttr = new RegExp('translate="I18N_', 'g');
```

Comment:
I put the name in capital letters to emphasize they are constants, I didn't declare them as cons because it says it doesn't work well with IE. Is it ok?